docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -1443,8 +1445,8 @@ Identifiers
     /* Alias Identification declaration (the "u" of "FROM User u") */
     AliasIdentificationVariable :: = identifier
 
-    /* identifier that must be a class name (the "User" of "FROM User u") */
-    AbstractSchemaName ::= identifier
+    /* identifier that must be a class name (the "User" of "FROM User u"), possibly as a fully qualified class name or namespace-aliased */
+    AbstractSchemaName ::= fully_qualified_name | aliased_name | identifier
```

Comment:
`fully_qualified_name` and `aliased_name` are not terminals. Keep the `identifier` (which holds both of your conceptualized values) only